src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -204,6 +207,10 @@ public void progress() {
 					final Location loc = this.player.getEyeLocation().add(this.player.getEyeLocation().getDirection().normalize().multiply(1.2));
 					loc.add(0, 0.3, 0);
 					playLightningbendingParticle(loc, 0.2F, 0.2F, 0.2F);
+					if(ThreadLocalRandom.current().nextDouble() < .2){

```

Comment:
There should be a space between the if and opening parenthesis. Additionally, there should be a space between the closing parenthesis and the opening brace.

```java
if (ThreadLocalRandom.current().nextDouble() < .2) {
```